Sources/Database/Database.swift

```diff
@@ -276,7 +278,9 @@ private struct _Client {
         ])
 
       return subscription.flatMap { subscription in
-        self.execute(
+        guard isOwnerTakingSeat else { return pure(subscription) }
```

Comment:
and here is where we implement that logic in the database. 

it kinda seems weird to have a single database endpoint do two queries with logic tbh. maybe this should be split up.